includes/controller/StoreAuthDataController.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace BitCode\FI\controller;
+
+use BitCode\FI\Core\Database\AuthModel;
+
+final class StoreAuthDataController
```

Comment:
why this class name in StoreAuthDataController, is this class only used for store data?